TACC/Core-CMS
# task/tup-681 Turn Off Drop Cap for Single Article

## Overview

Added in a style to turn off drop caps when a tag is added to an article.

## Related

- [TUP-681](https://tacc-main.atlassian.net/browse/TUP-681)

## Changes

Adds in a style to hide drop caps when tag is added to article

## Testing

Make sure you're logged into the portal.

1. Add the tag No Drop Cap to tags
   1. User Settings
   2. Tags on the left side nav
   3. Add Tag
   4. Name it No Drop Cap and make sure the meta tag is no-drop-cap
2. Add the tag to the article
   1. Create the article
   2. Go into editing the article
   3. Double click the article header
   4. Show the info section
   5. Add in the tag that you made in step 1.

Side Note: Make sure to add in and remove the tag to make sure it's effecting the drop cap.

## UI

| Without the tag | With the tag |
| --- | --- |
| <img width="2560" alt="Screenshot 2024-01-17 at 12 44 59 PM" src="https://github.com/TACC/Core-CMS/assets/63771558/a8e1e69b-3445-4669-8081-a20356b42a8f"> | <img width="2560" alt="Screenshot 2024-01-17 at 12 46 29 PM" src="https://github.com/TACC/Core-CMS/assets/63771558/3a8b2b68-450d-490b-9241-efcb25e5c0f4"> |
